nmpolicy/internal/resolver/resolver_test.go

```diff
@@ -97,8 +108,16 @@ func TestFilter(t *testing.T) {
 		testFilterMapListOnSecondPathIdentity(t)
 		testFilterMapListOnFirstPathIdentity(t)
 		testFilterList(t)
+		testFilterCaptureRef(t)
```

Comment:
This would be the same as `testFilterCaptureRefNotFound` since it's not resolved it's not present at capturedStates, and since it's not at the `captureASTPool` either it will not try to evaluate.

Reply:
I meant something that is in the `captureASTPool`.